Backend/app/rider-platform/rider-app/Main/src/API/UI/Cancel.hs

```diff
@@ -33,6 +34,13 @@ import qualified SharedLogic.CallBPP as CallBPP
 import Tools.Auth
 
 type API =
+  CancelAPI
```

Comment:
@sidsethu Are we not making it part of issue management? If yes, they we don't need separate API for this.

Reply:
Issue management is not live yet right?